pkg/utils/errors.go

```diff
@@ -0,0 +1,11 @@
+package utils
+
+import "fmt"
+
+func ErrorIfOccurrs(err error, message string, args ...interface{}) error {
```

Comment:
```suggestion
func ErrorIfOccurs(err error, message string, args ...interface{}) error {
```

Reply:
Right, I initially named it ErrorIfOccurred which made sense but renamed it to "If occurs" and forgot the double r :)